client/src/components/Habits/HabitsView.js

```diff
@@ -109,6 +112,72 @@ const HabitsView = () => {
             value={name}
             onChange={(e) => setName(e.target.value)}
           />
+          <TimePicker label="start-time" disableOpenPicker onChange={(newStart) => setStartTime(newStart)}/>
+          <TimePicker label="end-time" disableOpenPicker onChange={(newEnd) => setEndTime(newEnd)}/>
+          <br></br>
+          <label htmlFor="sunday">
+            <Checkbox
+              id="sunday" 
+              defaultChecked
+              icon={"S"}
+              ripple={false}
+              className="w-8 h-8"
+              onChange={() => handleDays(0)}/>
+          </label>
+          <label htmlFor="monday">
+            <Checkbox
```

Comment:
nit for later on if we have time/effort we can refactor to use lists to generate these for less redudant code